templates/triage/pulls.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <style>
+            * { font-family: sans-serif; }
+            h1 { font-size: 20px; }
+            h2 { font-size: 16px; }
+            p { font-size: 15px; }
+            li { font-size: 13px; }
+
+            table { border-collapse: collapse; }
+            td, th { border-bottom: 1px solid #ddd; padding: 5px 6px; font-size: 13px; text-align: left; vertical-align: baseline; }
+            tr:nth-child(even) { background: #eee; }
+
+            .dot:before {
+                content: "";
+                display: inline-block;
+                width: 0.4em;
+                height: 0.4em;
+                border-radius: 50% 50%;
+                margin-right: 0.3em;
+                border: 1px solid transparent;
+            }
+            .need-triage-red:before {
+                background: #FF0000;
+                border-color: black;
+                border-radius:2;
+            }
+            .need-triage-yellow:before {
+                background: #FFFF00;
+                border-color: black;
+                border-radius:2;
+            }
+            .need-triage-green:before {
+                background: #90EE90;
+                border-color: black;
+                border-radius:2;
+            }
+        </style>
+    </head>
+
+    <body>
+        <h1>Toriage - <a href="https://github.com/{{ owner }}/{{ repo }}">{{ owner }}/{{ repo }}</a></h1>
+        <table>
+            <thead>
+                <tr>
+                    <th>#</th>
+                    <th>Need triage</th>
+                    <th>Wait for</th>
+                    <th>Title</th>
+                    <th>Author</th>
+                    <th>Assignee</th>
+                    <th>Labels</th>
+                    <th>Updated at</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for pull in pulls %}
+                    <tr>
+                        <td><a href="{{ pull.html_url }}">{{ pull.number }}</a></td>
+                        <td class='dot need-triage-{{ pull.need_triage }}'>{{ pull.days_from_last_updated_at }} {% if pull.days_from_last_updated_at > 1 %}days{% else %}day{% endif %}</td>
+                        <td>{% if pull.wait_for_author %}<b>{{ pull.author }}</b>{% elif pull.wait_for_review %}<b>{{ pull.assignee }}</b>{% endif %}</td>
+                        <td>{{ pull.title }}</td>
+                        <td {% if pull.wait_for_author %} style='font-weight: bold;'{% endif %}>{{ pull.author }}</td>
+                        <td {% if pull.wait_for_review %} style='font-weight: bold;'{% endif %}>{{ pull.assignee }}</td>
```

Comment:
Thanks for your review!

It's because we can faster spot whether `Wait for` is author or reviewer. I implemented this from [this feedback](https://rust-lang.zulipchat.com/#narrow/stream/257328-clippy/topic/can.20we.20automate.20triaging.20PRs.20a.20bit.3F/near/227437233).

Reply:
Oh I see, then a note is definitely helpful :)

Reply:
Thanks, I added a comment as a note!